packages/foundation/src/atoms/core/colors/brand.css

```diff
@@ -1,38 +1,38 @@
-html * {
+.o-ui-theme {
```

Comment:
Since foundation is all about token I am not sure about naming the scope "o-ui-theme"